building-with-ocean/using-ocean-libraries/configuration.md

```diff
@@ -0,0 +1,199 @@
+# Configuration
+
+### Obtaining API key for Ethereum node provider
+
+Ocean libraries need an Ethereum node provider API key to send transactions to the Ocean Protocol's Smart contracts.  See this [guide](../obtaining-api-key.md) on getting an API key to interact with EVM compatible networks. The supported networks are listed [here](../../core-concepts/networks.md).
```

Comment:
Rather than sending them to the "Obtaining Ethereum node provider API key" page, let's just move that content here. It's not that long